src/main/java/com/gmail/mrphpfan/mccombatlevel/PlayerCombatLevelChangeEvent.java

```diff
@@ -0,0 +1,77 @@
+package com.gmail.mrphpfan.mccombatlevel;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+
+/**
+ * Called when a players combat level changes
+ * 
+ * @author Zettelkasten
+ *
+ */
+public class PlayerCombatLevelChangeEvent extends PlayerEvent implements Cancellable {
+
+	private final static HandlerList handlers = new HandlerList();
+	
+	private final int oldLevel;
+	private int newLevel;
+	
+	private boolean cancel;
+
+	public PlayerCombatLevelChangeEvent(Player player, int oldLevel, int newLevel) {
+		super(player);
+
+		this.oldLevel = oldLevel;
+		this.newLevel = newLevel;
+		
+		this.cancel = false;
```

Comment:
Ah, sorry. I'm not used to GitHub. I like to write that line as it explicitly states that the event is not canceled by default. It's there for the sake of completion and that way I know I have not forgot anything. I will not mind if you remove it though :)